packages/core/src/service/helpers/slug-validator/slug-validator.ts

```diff
@@ -29,7 +30,7 @@ export type TranslationEntity = VendureEntity & {
     id: ID;
     languageCode: LanguageCode;
     slug: string;
-    base: any;
+    base: Collection | Product;
```

Comment:
This would cause an issue if someone wanted to use this class in an external plugin, where they might have defined a custom entity which follows the same pattern. It would in fact be a breaking change.

Perhaps there is a way to express this more generally, but I'd say spending a lot of time getting a type perfectly right here is not high priority, since we only actually reference it here: https://github.com/vendure-ecommerce/vendure/blob/834c05c514ad2abfa94f56afa0560c3e205ea2ca/packages/core/src/service/helpers/slug-validator/slug-validator.ts#L81

Reply:
Good point. For some reason, I thought slugvalidator was only used internally. Reverted :ok_hand: